### ISSUE TITLE
Won't work on Windows

### ISSUE THREAD

**Issue by BramVanroy** [robertostling/eflomal#6]
I was hoping there was an aligner that actually worked on Windows, but unfortunately it doesn't. The first issue I bump into is the file handling.

On Windows you can't re-open `NamedTemporaryFile  `files that are open - it will give a Permission Denied error. It is bad practice either way.

This happens quite often and the actual idea behind the current design is not clear to me.  The `read_text` and `write_text` items should be re-written to be consistent with handling files. Either open a given URL, or pass an open file handle (that is not opened again in the function).

**Comment by Nouran-Khallaf** [comment]
i am facing the same problem did you find a solution?

**Comment by BramVanroy** [comment]
Alas, no. Did not get any feedback from authors.

**Comment by robertostling** [comment]
Hi, sorry I missed this. Unfortunately I don't have a Windows system to test on, so I can't debug this on my own. If anyone comes up with a portable solution I would be happy to accept it as a pull request. An emergency measure would be to add an option that creates a temporary file in a less secure way. Would that be acceptable for your use case?

**Comment by Nouran-Khallaf** [comment]
yes, I tried to add a specific location for the temp files but it did not work, please if you have any solution to specify the place it will be great. Many thanks

**Comment by TommiNieminen** [comment]
This is apparently a difference between Unix and Windows, Windows doesn't allow you to open a NamedTemporaryFile again. As a workaround, you can add "delete=False" to the NamedTemporaryFiles calls and close the files before align function is called. This does mean the temp files aren't deleted, so you'll also need to add temp file cleanup after the align call. I'm experimenting with eflomal for use with a Windows-based tool, so I might be able to make the necessary changes and add them to the codebase later.
